src/core/constants.js

```diff
@@ -140,7 +140,8 @@ export const SEARCH_SORT_UPDATED = 'updated';
 export const COLLECTION_SORT_DATE_ADDED_ASCENDING = 'added';
 export const COLLECTION_SORT_DATE_ADDED_DESCENDING = '-added';
 export const COLLECTION_SORT_NAME = 'name';
-export const COLLECTION_SORT_POPULARITY = 'popularity';
+// Note that popularity sort should be in descending order.
+export const COLLECTION_SORT_POPULARITY = '-popularity';
```

Comment:
Can we suffix this constant with `_DESCENDING` to make things explicit (and avoid the comment here)?

Reply:
Good idea, thanks!